contributors/design-proposals/high-availability.md

```diff
@@ -1,7 +1,7 @@
 # High Availability of Scheduling and Controller Components in Kubernetes
 
 This document is deprecated. For more details about running a highly available
-cluster master, please see the [admin instructions document](https://github.com/kubernetes/kubernetes/blob/master/docs/admin/high-availability.md).
+cluster master, please see the [admin instructions document](https://github.com/kubernetes/kubernetes.github.io/tree/master/docs/admin/high-availability).
```

Comment:
It would be good to use this link instead: 
https://kubernetes.io/docs/admin/high-availability/